From 9a8302965354b3a609d33cabf2b970c327166f0a Mon Sep 17 00:00:00 2001
From: Gwendal Grignou <gwendal@chromium.org>
Date: Tue, 6 Feb 2018 10:29:23 -0800
Subject: [PATCH] Fix is_capable

udevadmn is now in /bin since CL:568592

BUG=chromium:809225
TEST=trybot on fizz, messages: """laptop-mode[1549]: failed - udev not
active?""" gone.

Signed-off-by: Gwendal Grignou <gwendal@chromium.org>
---
 usr/share/laptop-mode-tools/modules/hdparm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- laptop-mode-tools-1.59/usr/share/laptop-mode-tools/modules/hdparm
+++ laptop-mode-tools-1.59/usr/share/laptop-mode-tools/modules/hdparm.new
@@ -34,7 +34,7 @@ is_capable() {
 	# It assumes more or less recent udev (> 070)
 	if [ $HAVE_UDEVINFO -ne 0 ] ; then
 		log "VERBOSE" "Querying $1 media type using udevinfo: "
-		if [ -x /sbin/udevadm ]; then
+		if [ -x "$(which udevadm 2> /dev/null)" ]; then
 			eval "$(udevadm info -q env -n $1 | egrep '(ID_TYPE=|ID_BUS=)' )"
 		else
 			eval "$(udevinfo -q env -n $1 | egrep '(ID_TYPE=|ID_BUS=)' )"
-- 
2.13.5

